### ISSUE TITLE
Audio filler preparation fails

### ISSUE THREAD

**Issue by thomaskhub** [thomaskhub/ums#8]
__What is the commit hash you checked out__
06c7c23c214a82cdbfc944c2fde3fd8cb2a064d8

__What did you do__
Just started ums miltiple times. 

__What si the issue__
This error is displayed: 

```bash 
inputSwitch::could not prepare audio filler
```

**Comment by thomaskhub** [comment]
Issue is that:
```c
prepareAudioFiller()
```

does not return anything but we checking the return value. So sometime application just returns values unequal 0 and causes 
the error. Fix is to return 0 on success. 

implemented in branch: bugfix/#8-prepare-audio-filler

**Comment by thomaskhub** [comment]
merged into master
